utils/hostname.py

```diff
@@ -8,7 +8,7 @@
 import socket
 
 # project
-from utils.cloud_metadata import EC2, GCE
+from utils.cloud_metadata import EC2, GCE, Azure
```

Comment:
you should remove the import from here and add it to `checks/collector.py` (CI is failing on this)